roles/ovirt-host-deploy-ovirt-metrics/tasks/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+- name: Run oVirt metrics to configure metrics and logs collection on the host
+  command: /usr/share/ovirt-engine-metrics/setup/ansible/configure_ovirt_machines_for_metrics.sh --scope={{ inventory_hostname }}
```

Comment:
Where is this file located on the host or engine?

Reply:
On the engine machine

Reply:
Ok, then I think this will work good. The only issue could be dev-env, people need to make sure to have metrics installed always.

Reply:
Actually they don't. If a metric store is configured then it will configure the host. Otherwise, it will show a message and end.

Reply:
Well, this task will fail, because the command is not found, and so the host install will report `install_failed` status because ansible-playbook command will return return code `2`.